.storybook/overrides.css

```diff
@@ -1,3 +1,13 @@
+:root,
+.theme-light {
+  --color-template-surface-background-page: #ffffff00;
+}
+
+:host,
+.theme-dark {
+  --color-template-surface-background-page: #141414;
+}
```

Comment:
There are extended styles for this. But they're only used by Storybook. It wasn't worth exposing these to the world just for this one case, in my opinion. Let me know if you feel differently.